.github/workflows/ci.yml

```diff
@@ -46,3 +46,34 @@ jobs:
         run: >
           python format_checker/main.py ${{github.event.before}}
           ${{github.event.after}}
+  Maven:
+    on:
+    schedule:
+      - cron: '40 10 1 * *'
+      
+    -name: Maven Update pr-data.csv
+    runs-on: ubuntu-latest
+     
+      steps:
```

Comment:
This line is not properly indented. Please ensure that your changes pass the checks https://github.com/TestingResearchIllinois/idoft/pull/52/checks